pkg/peggy/lib/version.js

```diff
@@ -0,0 +1,3 @@
+"use strict";
+
+module.exports = "1.0.0";
```

Comment:
it's probably best to generate this from the root package

i have standard stuff for this that i can donate if youw ant it

Reply:
yes please.

Reply:
once we settle this down so you have a clean place to PR from.